Formula/solc-select.rb

```diff
@@ -38,6 +39,11 @@ def install
     system bin/"solc-select", "install", "0.8.0"
     system bin/"solc-select", "use", "0.5.7"
 
+    assert_match(/0\.5\.7.*current/, shell_output("#{bin}/solc-select versions"))
+
+    # running solc itself requires an Intel system or Rosetta
+    return if Hardware::CPU.arm?
+
```

Comment:
That seems like it won't be very useful for people to have this bottled on ARM then. I'd say it's better to mark it as x86.

Reply:
Hi @SMillerDev! The software itself (`solc-select`) is written in Python and can run fine on ARM. It's also a runtime dependency for some of the other programs in this PR, which can be meaningfully used on an ARM Mac without Rosetta. My understanding is that marking `solc-select` as x86 would mean none of the packages here get bottled, which would not be desirable.